### DISCUSSION TITLE
群晖docker安装最新版后 启动不了代理，报错如下！麻烦哪位大神能解答一下

### DISCUSSION THREAD

**Discussion by lwf540** [v2rayA/v2rayA#1435] [Q&A]
failed to start v2ray-core: not support "redirect" mode of transparent proxy: ExecCommands: iptables-legacy -w 2 -t nat -N TP_OUT iptables v1.8.10 (nf_tables): Could not fetch rule set generation id: Invalid argument : exit status 4

**Comment by guyqqcom** [comment]
同问

**Comment by minzhengtc** [comment]
我也是，怎么解决呀

**Comment by milkpodd** [comment]
用下面的docker-compose配置
services:
  v2raya:
    image: mzz2017/v2raya:v2.2.6.3
    container_name: v2raya
    privileged: true
    restart: always
    ports:
      - "2017:2017"
      - "20170-20173:20170-20173"
    environment:
      V2RAYA_LOG_FILE: "/tmp/v2raya.log"
      V2RAYA_V2RAY_BIN: "/usr/local/bin/xray"
      IPTABLES_MODE: "legacy"
      TZ: "Asia/Shanghai"
    volumes:
      - "/lib/modules:/lib/modules:ro"
      - "/volume1/docker/v2raya/data:/etc/v2raya"

**Comment by zzpkids** [comment]
<img width="1360" alt="截屏2025-01-23 11 51 02" src="https://github.com/user-attachments/assets/32db7e98-2131-4628-b538-151d122a7111" />
和你一样的问题，但是我是unraid docker,现在参考楼上大佬，如图手动配置解决了，不会报错了
